include/ignition/gazebo/detail/ComponentStorageBase.hh

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright (C) 2021 Open Source Robotics Foundation
```

Reply:
This file is being restored because #856 was deleting it, it's not new :wink: